llvm/include/llvm/MC/CAS/MCCASObjectV1.h

```diff
@@ -346,6 +346,9 @@ class MCCASBuilder {
   Error createStringSection(StringRef S,
                             std::function<Error(StringRef)> CreateFn);
 
+  Error createLineSection(StringRef S,
```

Comment:
I think it would be nice to add a Doxygen comment for the function.
Should we call it `createDebugLineSection` for more clarity?

Reply:
Agree.

Also it might just be better the CreateFn is `Error(ArrayRef<char>)`? You can slice ArrayRef and don't need that StringRef conversion?

Reply:
I think the challenge is that we cannot create a `BinaryStreamReader` with an `ArrayRef<char>`, only `ArrayRef<uint8_t>` and `StringRef`

Reply:
Yep, I need the `StringRef` to convert it to a `BinaryStreamReader` which will take care of slicing with endianness in mind